docs/UserGuide.adoc

```diff
@@ -622,42 +622,44 @@ Format: `exit`
 // tag::calendar[]
 === Using the calendar
 
-A calendar is useful for marking the important dates. When you have many deadlines and birthdays to take note, the calendar allows you to view all these important dates at a glance. +
+A calendar is useful for marking important dates. When you have many deadlines and birthdays to take note of, the calendar allows you to view all these important dates at a glance. +
 
 ==== Marking the dates
 
-With many contacts and tasks to manage, differentiating the birthdays and deadlines on the calendar can be done with the use of colours.
+With many contacts and tasks to manage, differentiating birthdays and deadlines on the calendar can be done with the help of colours.
 
 [NOTE]
-The marking of date is done automatically when a contact/task is added, edited or deleted.
+Dates are marked automatically when a contact/task is added, edited or deleted.
 
 Examples:
 
-* A task has deadline on `17-07-2017`. +
+* A task has a deadline on `17-07-2017`. +
 The corresponding date on the calendar is marked *red*. +
-* A contact has birthday on `09-12-2017`. +
+* A contact has his/her birthday on `09-12-2017`. +
 The corresponding date on the calendar is marked *pink*. +
-* `17-07-2017` is a date that is a task's deadline and a contact's birthday. +
+* `17-07-2017` that is both a task's deadline and a contact's birthday. +
```

Comment:
17-07-2017 is both a task's deadline...*
Would recommend you change the date format to 17th July 2017.